### PR TITLE
Add Az Module 2.8.0 to installation scripts

### PR DESCRIPTION
**Pull request by Lutando**
As per https://github.com/microsoft/azure-pipelines-image-generation/issues/1359 I have added Az 2.8.0 to the image installation scripts.

If there is anything else missing let me know

### CODE REVIEW

File: images/linux/Ubuntu1604-README.md [hunk 1]
```diff
@@ -228,6 +228,7 @@ the
 - Az Module (1.6.0)
 - Az Module (2.3.2)
 - Az Module (2.6.0)
+- Az Module (2.8.0)
```
Comment (alepauly) [inline images/linux/Ubuntu1604-README.md]: Please don't change the readme files, those should get autogenerated  (via the DocumentInstalledItem calls) when we build the images.

File: images/linux/scripts/installers/1604/azpowershell.sh [hunk 2]
```diff
@@ -13,6 +13,7 @@ sudo pwsh -Command 'Save-Module -Name Az -LiteralPath /usr/share/az_1.0.0 -Requi
 sudo pwsh -Command 'Save-Module -Name Az -LiteralPath /usr/share/az_1.6.0 -RequiredVersion 1.6.0 -Force'
 sudo pwsh -Command 'Save-Module -Name Az -LiteralPath /usr/share/az_2.3.2 -RequiredVersion 2.3.2 -Force'
 sudo pwsh -Command 'Save-Module -Name Az -LiteralPath /usr/share/az_2.6.0 -RequiredVersion 2.6.0 -Force'
+sudo pwsh -Command 'Save-Module -Name Az -LiteralPath /usr/share/az_2.8.0 -RequiredVersion 2.8.0 -Force'
```
Comment (niadak) [inline images/linux/scripts/installers/1604/azpowershell.sh]: Can we add the latest version [Az 3.1.0](https://github.com/Azure/azure-powershell/releases/tag/v3.1.0-November2019) directly?

Comment (Lutando) [reply images/linux/scripts/installers/1604/azpowershell.sh]: @niadak in another PR? I think 3.x.x breaks somethings for the 2.x.x people

File: images/linux/Ubuntu1604-README.md (hunk 1 above)
Comment (Lutando) [reply images/linux/Ubuntu1604-README.md]: gotcha

File: images/linux/scripts/installers/1604/azpowershell.sh (hunk 2 above)
Comment (niadak) [reply images/linux/scripts/installers/1604/azpowershell.sh]: Sure.

**Comment by alepauly** [comment]
@Lutando if you can resolve the conflicts that just showed up I'll go ahead and merge this. Thanks again!

**Comment by Lutando** [comment]
@alepauly looks like the files that I wanted to do a PR on have been moved to https://github.com/actions/virtual-environments. On this repository they have been effectively deleted.

eg - https://github.com/actions/virtual-environments/blob/7463ea032be0ad4b38dc7a88ebd5375880a53f7f/images/linux/scripts/installers/1604/azpowershell.sh

This PR is now essentially irrelevant  I will have to re initiate one on the other repository now.

Sorry for this, you can close this PR

**Comment by alepauly** [comment]
> 
> 
> @alepauly looks like the files that I wanted to do a PR on have been moved to https://github.com/actions/virtual-environments. On this repository they have been effectively deleted.
> 
> eg - https://github.com/actions/virtual-environments/blob/7463ea032be0ad4b38dc7a88ebd5375880a53f7f/images/linux/scripts/installers/1604/azpowershell.sh
> 
> This PR is now essentially irrelevant I will have to re initiate one on the other repository now.
> 
> Sorry for this, you can close this PR

Oh that's right. I knew that, just wasn't thinking. Yeah please open one on the other repo and I'll approve and merge. Thanks, and sorry for the inconvenience!

**Comment by alepauly** [comment]
This has been merge in actions/virtual-environments

**Pull request closed without merging**